Content.Client/Atmos/Monitor/AtmosMonitorVisualizer.cs

```diff
@@ -26,13 +26,15 @@ public sealed class AtmosMonitorVisualizer : AppearanceVisualizer
         [DataField("setOnDepowered")]
         private readonly Dictionary<string, string>? _setOnDepowered;
 
+        [Obsolete("Subscribe to your component being initialised instead.")]
```

Comment:
Are these not going to be changed? If they are, is there another place that says to change them like an issue?

Reply:
this is a good point: i think these should at least have a link to the instructions for the AppearanceVisualizer to VisualizerSystem<T> port: https://docs.spacestation14.io/en/engine/porting-visualizers

Reply:
The method they are overriding are obsolete, therefore these should be obsolete as well. It gives a compiler warning, which is why I am adding Obsolete attributes here.